src/components/mermaidInputs/InputWithLabelAndValidation/InputWithLabelAndValidation.js

```diff
@@ -1,28 +1,21 @@
 import React, { useRef, useState } from 'react'
 import PropTypes from 'prop-types'
-import styled from 'styled-components'
 
-import { Input, InputRow, HelperText, LabelContainer, RequiredIndicator } from '../../generic/form'
+import {
+  Input,
+  InputContainer,
+  InputRow,
+  HelperText,
+  LabelContainer,
+  RequiredIndicator,
+} from '../../generic/form'
 import { useStopInputScrollingIncrementNumber } from '../../../library/useStopInputScrollingIncrementNumber'
 import InputNumberNoScrollWithUnit from '../../generic/InputNumberNoScrollWithUnit'
 
 import InputValidationInfo from '../InputValidationInfo/InputValidationInfo'
 import mermaidInputsPropTypes from '../mermaidInputsPropTypes'
-import { IconButton } from '../../generic/buttons'
+import { IconButton, CheckBoxLabel, InputButton } from '../../generic/buttons'
 import { IconInfo } from '../../icons'
-import theme from '../../../theme'
-
-const CheckBoxLabel = styled.label`
```

Comment:
I dont see this component being used generically. Is it safe to add to the generics or is it specific to this `InputWithLabelAndValidation` use case?

Reply:
it's only being used in one instance now - but recently we've been adding extras to inputs (checkboxes + buttons). I sense there will be more cases like this added in the future